Fix issue #12652 - Sidebar spacing

geekpulp opened the pull request:
Fix spacing on sidebar

Adding some spacing on the sidebar below each of the descriptions. This
corrects issue #12652

## Description
<!-- Please describe what you have changed or added -->
- remove margin from .components-base-control__help and created margin in .components-base-control + .components-base-control.
- moved from px to $grid-size

## How has this been tested?
<!-- Please describe in detail how you tested your changes. -->
<!-- Include details of your testing environment, tests ran to see how -->
<!-- your change affects other areas of the code, etc. -->
Minimal testing, made the change, tried it in safari, Chrome, Firefox on MacOS 
## Screenshots <!-- if applicable -->

## Types of changes
<!-- What types of changes does your code introduce?  --> small css change that increase spacing on sidebar below description

<!-- Bug fix (non-breaking change which fixes an issue) -->
<!-- New feature (non-breaking change which adds functionality) -->
<!-- Breaking change (fix or feature that would cause existing functionality to not work as expected) -->

## Checklist:
- [X ] My code is tested.
- [ X] My code follows the WordPress code style. <!-- Check code: `npm run lint`, Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/ -->
- [ X] My code follows the accessibility standards. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/accessibility-coding-standards/ -->
- [ X] My code has proper inline documentation. <!-- Guidelines: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/ -->
- [ X] I've included developer documentation if appropriate. <!-- Handbook: https://wordpress.org/gutenberg/handbook/designers-developers/ -->

geekpulp committed:
Fix issue #12652 - Sidebar spacing

Fix spacing on sidebar

Adding some spacing on the sidebar below each of the descriptions. This
corrects issue #12652

youknowriad commented on packages/components/src/base-control/style.scss:
```diff
@@ -18,6 +18,6 @@
 		margin-top: -$grid-size;
 		font-style: italic;
-		margin-bottom: 0;
+		margin-bottom: 20px;
```
Thanks for the PR @geekpulp 

I'm wondering if the best way to fix is to add a margin bottom to the help component (which may or may not be present depending on what component is used). It feels like the root issue is that there's no space between controls. So maybe the fix is more to target something like `.components-base-control + .components-base-control`

What do you think?

youknowriad replied:
Another thing is I think we're trying to make sure we use the $grid-size as a basis for all these spaces in the UI. So maybe `$grid-size*2` or something? 

cc @jasmussen

geekpulp replied:
Thanks for the feedback. You’re right. I’ll change my approach and update.

geekpulp replied:
@youknowriad Ok, I've had another crack at it. Seems to have done the trick. Also updated a related stylesheet to use $grid-size.

It's quite a learning curve but this is strangely fun.

jasmussen replied:
Thank you for using $grid-size! If you need to, there's also $grid-size-large, which is the same as $grid-size * 2.

geekpulp replied:
@jasmussen Cool, I'll change that then. Is there somewhere I can go to see standards like grid-size so I can proactively make changes like that? Is it something documented somewhere?

geekpulp replied:
Hmm showing my newbie skills here with Git. Sorting it.

geekpulp committed:
Update margin styles

remove margin from .components-base-control__help and created margin in .components-base-control + .components-base-control.

This should make the spacing between components in the sidebar more appealing.

geekpulp committed:
Update margin sizes

Changing to $gridsize from px

geekpulp committed:
update to grid-size-large

Change from grid-size * 2 to grid-size-large

geekpulp committed:
Merge branch 'master' of https://github.com/WordPress/gutenberg

geekpulp committed:
Chnage PollDaddy to Crowdsignal

geekpulp committed:
Merge branch 'master' of https://github.com/geekpulp/gutenberg

* 'master' of https://github.com/geekpulp/gutenberg:
  update to grid-size-large
